File: PooCC01CNetz/Semana04/Java/Ejercicios01/Ciudad.java
public class Ciudad{
    public static void main(String[] args) {
        Domicilio dom = new Domicilio();
        dom.setCalle("Av. monte de las cruces");
        dom.setNum(14);
        dom.setCp(5760);

        System.out.println("Calle   : " + dom.getCalle());
        System.out.println("Numero  : " + dom.getNum());
        System.out.println("CP      : " + dom.getCp());

        Ciudadano c1 = new Ciudadano();
        c1.setNombre("Alejandra");
        c1.setDomicilio(dom);

        System.out.println("\nNombre  : " + c1.getNombre());
        System.out.println("Calle   : " + c1.getDomicilio().getCalle());
        System.out.println("Numero  : " + c1.getDomicilio().getNum());
        System.out.println("CP      : " + c1.getDomicilio().getCp());
    }
}

File: PooCC01CNetz/Semana04/Python/Ejercicios01/Ciudad.py
from Ciudadano import *
class Ciudad:
    def main():
        dom = Domicilio()
        dom.setCalle("Av. monte de las cruces")
        dom.setNum(14)
        dom.setCp(5760)

        print("Calle   : " + dom.getCalle())
        print("Numero  : " + str(dom.getNum()))
        print("CP      : " + str(dom.getCp()))

        c1 = Ciudadano()
        c1.setNombre("Alejandra")
        c1.setDomicilio(dom)

        print("\nNombre  : " + c1.getNombre())
        print("Calle   : " + c1.getDomicilio().getCalle())
        print("Numero  : " + str(c1.getDomicilio().getNum()))
        print("CP      : " + str(c1.getDomicilio().getCp()))


    main()